docs/en/samples/index.md

```diff
@@ -96,54 +95,60 @@ headers = {
 
 #### Constructing a JSON command
 
-The body of the request contains the JSON command structure. First, we create the JSON array:
+The body of the request is a list of JSON objects. You can add maximum 10 such objects in this list. Sample  for 
 
 ```python
-json_data = \
+body = \
 [
   {
-    "user" : "john.doe@" + domain,
+    "user" : "john.doe@domain.com",
     "do" : [
       {
         "addAdobeID" : {
-          "email" : "john.doe@" + domain
+          "email" : "john.doe@domain.com",
+          "firstname": "John",
+          "lastname": "Doe",
+          "country": "US"
+          "option": "ignoreIfAlreadyExists"
         }
       }
     ]
   }
 ]
 ```
 
-Next, we convert the JSON array to a string to be included in the request body:
-
-```python
-# prepare body
-body = json.dumps(json_data)
-```
-
 #### Making a request
 
 Finally, we connect to the server, send the request we have created, and receive a response:
 
 ```python
 # send http request
-res = requests.post(url, headers=headers, data=body)
+r = requests.request(method, url, data=body, headers=headers)
```

Comment:
```suggestion
res = requests.post(url, headers=headers, data=body)
```
I don't think these changes are necessary.

Reply:
I did not create these new samples looking at the old ones; these were created from 0 and they reflect variable names and function definitions that follow same pattern in all my files. The walkthrough was modified based on how these scripts should be used, in the idea that they are all standalone scripts, isolated from each other for the single purpose of demonstrating that single piece of functionality.